src/components/List/AddCard/AddCardForm/AddCardForm.js

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useRef, useState } from "react";
+import { FiX } from "react-icons/fi";
+import { useDispatch } from "react-redux";
+import { addCard } from "../../../../features/cardsSlice";
+
+const AddCardForm = ({ setOpen, open, listId }) => {
+  const dispatch = useDispatch();
+  const [text, setText] = useState("");
+  const inputRef = useRef(null);
+  const inlineStyle = open ? { height: inputRef.current?.scrollHeight } : { height: 0, overflow: "hidden" };
```

Comment:
This inlineStyle controls the visibility of the element. If it isnt open, the height is set to 0